.gitignore

```diff
@@ -0,0 +1,21 @@
+#local env files
+.env*
+.env.local
```

Comment:
The rule above excludes any file with the name that starts with ".env" so ".env.local" and ".env.production.local" rules are redundant.

Reply:
Ok, got it, I will fix it. Thanks for checking this Andrey!